cluster_management/src/test/java/com/pinterest/rocksplicator/codecs/WrappedDataThriftCodecTest.java

```diff
@@ -0,0 +1,97 @@
+package com.pinterest.rocksplicator.codecs;
```

Comment:
Also, are these tests run as part of the Travis CI?

Reply:
I don't think so. the travis yaml as defined is 

- cd ../cluster_management; mvn clean package -DskipTests # if you want to test:  mvn "-DfailIfNoTests=false" test
which will skip the tests. I tested them locally using mvn clean package in cluster_management directory. I also used local zk and performed writes and reads from zk using these codecs..